util/Migrator/DbScripts/2023-11-14_00_2019FamilyPlanPremium.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN TRY
+    BEGIN TRANSACTION;
+
+    UPDATE
+        [dbo].[Organization]
+    SET
+        [UsersGetPremium] = 0
+    WHERE
+        [PlanType] = 1 -- Families 2019 Annual
+
+    COMMIT TRANSACTION;
+END TRY
+BEGIN CATCH
+    ROLLBACK TRANSACTION;
+    THROW;
```

Comment:
@bitwarden/dept-devops what happens with our migrator when an error occurs? Does that get shared somewhere? And does this freeze the release?

Reply:
That's a good question, I expect that it would halt the step that it was on. Given the above comment, I'll go ahead with removing the transaction given that it's a single statement